src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -51,8 +51,6 @@
 from DIRAC.Core.Utilities import Network
 from DIRAC.Core.Utilities.JEncode import decode, encode
 
-from DIRAC.Resources.IdProvider.IdProviderFactory import IdProviderFactory
-from DIRAC.FrameworkSystem.private.authorization.utils.Tokens import getLocalTokenDict, writeTokenDictToTokenFile
```

Comment:
I don't think you need to hide it, I'd better try to rewrite it. The first thing that comes to mind is to describe these objects at the DIRAC.Core.Security level. what do you think about it?

Reply:
Honestly, it is very hard for me to give you an opinion. As you know, I spent quite some time reviewing what you did, but I did not go to that level of detail, so I can't give an educated answer.
I think though that until you fix it, hiding it there is a good trade off ?

Reply:
OK, I'll think about a possible solution later and do a PR or issue